classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] Please make sure when adding a student to a classroom it also sets the classroom for the student to achieve the `has-many` / `belongs-to` relationships.
  - You can consider the below code for this method.

```ruby
def add_student(student)
    @students.push(student)
    student.classroom = self
  end
```